python/ray/autoscaler/commands.py

```diff
@@ -920,8 +924,10 @@ def rsync(config_file: str,
                 # print rsync progress for single file rsync
                 cmd_output_util.set_output_redirected(False)
                 set_rsync_silent(False)
-
-                rsync(source, target)
+                try:
+                    rsync(source, target)
+                except:
+                    print(f"Rsync failed for node {node_id} \n")
             else:
                 updater.sync_file_mounts(rsync)
```

Comment:
Specifically, can you pas in the `exclude_list` into to the `rsync` function and the `updater.sync_file_mounts` function. 

Since `rsync` is just `updater.rsync_down` or `updater.rsync_up`, those two functions need to be modified.